salt/lax/config/opt-bot-lax-adaptor-app.cfg

```diff
@@ -20,6 +25,9 @@ cdn_iiif: {{ scheme }}://{{ pillar.elife.env }}--cdn-iiif.elifesciences.org/lax:
 {%- endif %}
 iiif: {{ scheme }}://{{ pillar.elife.env }}--iiif.elifesciences.org/lax:
 
+[glencoe]
+cache_requests: {{ pillar.elife.env not in ['prod', 'continuumtest'] }}
```

Comment:
`end2end` should be included with those two

Reply:
shouldn't this be a `pillar.lax.glencoe.cache_requests` value?

Reply:
I can make it configurable through pillar data

Reply:
updated. builder-config PR: https://github.com/elifesciences/builder-configuration/pull/75